Makefile

```diff
@@ -253,7 +253,7 @@ build-mo: $(MOFILES)
 .PHONY: static-analysis
 static-analysis:
 ifeq ($(shell command -v golangci-lint 2> /dev/null),)
-	go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.46.2
+	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $$HOME/go/bin
```

Comment:
I'd use `$(GOPATH)/bin` if possible.

Reply:
Better even is `$(go env GOPATH)`

Reply:
> I'd use `$(GOPATH)/bin` if possible.

You mean `${GOPATH}/bin` (curly braces)

Reply:
I tried `$(go env GOPATH)/bin` originally for microcloud but the runner didn't seem to like it. Now I'm thinking that was just a problem with my Makefile as it would have needed an extra `$`.

Reply:
The `Makefile` has some other occurrences of `$(GOPATH)/bin` (see the `update-schema` target).

Reply:
> The `Makefile` has some other occurrences of `$(GOPATH)/bin` (see the `update-schema` target).

You're right, I'm sorry. `$(GOPATH)` will work in a Makefile.

Reply:
It's not `$$(go env GOPATH)/bin` and working :tada: